extra_requirements/development.txt

```diff
@@ -32,3 +32,4 @@ pyahocorasick
 pytest
 pre-commit==3.7.0
 nbmake==1.5.3
+rdkit
```

Comment:
you use rdkit in production code, right? should be added to the other requirements.txt (please also use a fixed version)

Reply:
Agree!

Reply:
added